src/app/utils/account.ts

```diff
@@ -0,0 +1,9 @@
+import { staking } from '@oasisprotocol/client'
+
+export const addressToNumber = (address: string) => {
```

Comment:
This is specific to creating an account icon, so I would move it out of general utils.
The returned number is otherwise meaningless/arbitrary, not any kind of account number, and "seed" variable name comes from "a seed for randomization of icon shapes"